packages/scaffolding-cli/src/domain/model/prompt_answer.ts

```diff
@@ -5,6 +5,19 @@ export interface BusinessSection {
     component?: string
 }
 
+export const PlatformTypeEnum = {
+    AKS: 'aks',
+    GKE: 'gke',
+    EKS: 'eks',
+    ANY: 'any'
```

Comment:
Whats the Any for? Test only andn no deployment?

Reply:
yeah exactly we skip deployments for tests